api/v1/cluster_types.go

```diff
@@ -758,6 +758,25 @@ type BootstrapInitDB struct {
 	// after the cluster has been created - to be used with extreme care
 	// (by default empty)
 	PostInitTemplateSQL []string `json:"postInitTemplateSQL,omitempty"`
+
+	// PostInitApplicationSQLRefs points references to configmaps or secrets which
+	// contain SQL files, the general implementation order to these references is
+	// from all secrets to all configmaps, and inside secrets or configmaps,
+	// the implementation order is same as the order of each array
+	// (by default empty)
+	PostInitApplicationSQLRefs *PostInitApplicationSQLRefs `json:"postInitApplicationSQLRefs,omitempty"`
+}
+
+// PostInitApplicationSQLRefs points references to configmaps or secrets which
+// contain SQL files, the general implementation order to these references is
+// from all secrets to all configmaps, and inside secrets or configmaps,
+// the implementation order is same as the order of each array.
+type PostInitApplicationSQLRefs struct {
+	// SecretRefs holds a list of references to secrets.
+	SecretRefs []corev1.SecretKeySelector `json:"secretRefs,omitempty"`
+
+	// ConfigMapRefs holds a list of references to configmaps.
+	ConfigMapRefs []corev1.ConfigMapKeySelector `json:"configMapRefs,omitempty"`
```

Comment:
please use our own SecretKeySelector and ConfigMapKeySelector, see https://github.com/Anhui-tqhuang/cloudnative-pg/blob/main/api/v1/base_types.go#L32.